sql/auth/base/base_premium_auth_tables.sql

```diff
@@ -1,5 +1,6 @@
 CREATE TABLE IF NOT EXISTS `premium_account` (
   `account_id` int(10) unsigned NOT NULL,
   `premium_level` int(10) unsigned NOT NULL,
```

Comment:
```suggestion
  `premium_level` tinyint(3) unsigned NOT NULL,
```
no need for more than 255 levels

also it could be "signed" so we can use negative values (-128 to 127) too, what do you think?